nodeup/pkg/model/secrets.go

```diff
@@ -58,6 +58,26 @@ func (b *SecretBuilder) Build(c *fi.ModelBuilderContext) error {
 		c.AddTask(t)
 	}
 
+	if b.SecretStore != nil {
+		key := "nodedockercfg"
+		dockercfg, err := b.SecretStore.Secret(key)
+		if err != nil {
+			return err
+		}
+		if dockercfg == nil {
+			return fmt.Errorf("node docker config not found: %q", key)
```

Comment:
Is this an error?  Presumably not...  I suspect we'll fail e2e with this (but we'll see I guess!)